### PR TITLE
[SHELL32] Portuguese (PT) translation improvements and object adjustements

### PR DESCRIPTION
**Pull request by zecarlos1957**
[SHELL32] Improve Portuguese (PT) translation

[SHELLEXT] Improve Portuguese (PT) translation to acppage rc file

### CODE REVIEW

File: dll/win32/shell32/lang/pt-PT.rc [hunk 1]
```diff
@@ -182,8 +182,8 @@ BEGIN
     LTEXT "", -1, 8, 31, 223, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     LTEXT "Tipo de destino:", IDC_SHORTCUT_TYPE, 8, 38, 68, 10
     EDITTEXT IDC_SHORTCUT_TYPE_EDIT, 79, 37, 150, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "Localização do destino:", IDC_SHORTCUT_LOCATION, 8, 55, 68, 10
-    EDITTEXT IDC_SHORTCUT_LOCATION_EDIT, 79, 53, 150, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "Localização de destino:", IDC_SHORTCUT_LOCATION, 8, 50, 68, 15
```
Comment (HBelusca) [inline dll/win32/shell32/lang/pt-PT.rc:185]: Most of LTEXTs have a width of 10, except those that you have changed here (and below): why is it so?

Comment (zecarlos1957) [reply dll/win32/shell32/lang/pt-PT.rc:185]: these texts occupy two lines. So I had to increase the height

Comment (zecarlos1957) [reply dll/win32/shell32/lang/pt-PT.rc:185]: Before
We can't see all the tet

Comment (zecarlos1957) [reply dll/win32/shell32/lang/pt-PT.rc:185]: After
Aso in"Tipo de ficheiro"

Comment (HBelusca) [reply dll/win32/shell32/lang/pt-PT.rc:185]: ok thanks @zecarlos1957 I understand now :) But note, in the last screenshot, that there appears to be an alignment problem of the text zone "2,00 KB" with the other ones above it (the "189 bytes" and the "X:\Profiles\...").

Also, the livecd being ISO, i don't remember whether this stores any "file accessed date" value (if not, it defaults to "0" which means that 1st january 1601).

Comment (zecarlos1957) [reply dll/win32/shell32/lang/pt-PT.rc:185]: In the pt-PT.rc file all the edit boxes are aligned.
It seems that the problem lies in the file <drvdefext.cpp> by a space.
I will investigate and open a jira tiket

Comment (zecarlos1957) [reply dll/win32/shell32/lang/pt-PT.rc:185]: my mistake. My apologies. Found it

**Review by HBelusca — APPROVED** [review]

**Pull request merged**